test/JDBC/input/BABEL-4279-vu-prepare.mix

```diff
@@ -0,0 +1,48 @@
+-- tsql
+CREATE TABLE test_babel_4279_t1([ABC.nfds] INT, [DEf.j] INT);
+GO
+
+CREATE VIEW test_babel_4279_v1 AS SELECT test_babel_4279_t1.[ABC.nfds] from test_babel_4279_t1;
+GO
+
+CREATE VIEW test_babel_4279_v2 AS SELECT [test_babel_4279_t1].[ABC.nfds] ,test_babel_4279_t1.[DEf.j] from test_babel_4279_t1;
+GO
+
+CREATE DATABASE test_babel_4279_d1;
+GO
+
+USE test_babel_4279_d1;
+GO
+
+CREATE SCHEMA test_babel_4279_s1;
+GO
+
+CREATE TABLE test_babel_4279_s1.test_babel_4279_st1([ABC.nfds] INT, [DEf.j] INT);
+GO
+
+CREATE VIEW test_babel_4279_sv1 AS SELECT [test_babel_4279_s1].[test_babel_4279_st1].[ABC.nfds] from test_babel_4279_s1.test_babel_4279_st1;
+GO
+
+USE MASTER
+GO
+
+CREATE VIEW test_babel_4279_v3 AS SELECT [test_babel_4279_d1].[test_babel_4279_s1].[test_babel_4279_st1].[ABC.nfds] from [test_babel_4279_d1].[test_babel_4279_s1].[test_babel_4279_st1];
+GO
+
+CREATE TABLE test_babel_4279_t2(您您对您对您对您对您对您对您对您对您对您您您 INT, 对您对您对您对您对您对您对您 INT);
+GO
+
+CREATE VIEW test_babel_4279_v4 AS SELECT test_babel_4279_t2.[您您对您对您对您对您对您对您对您对您对您您您] from test_babel_4279_t2;
+GO
+
+CREATE VIEW test_babel_4279_v5 AS SELECT ぁあ.[您您对您对您对您对您对您对您对您对您对您您您] from test_babel_4279_t2 AS ぁあ;
+GO
+
+CREATE SCHEMA "tngdf'";
+GO
+
+CREATE TABLE "tngdf'".[sc,sdg"fdsngjds']("AB[C" INT);
```

Comment:
We should add tests for single quotes and no quotes as well.
Also, let's add tests for names with multiple double quotes, single quotes and square brackets at once. Something like following:
```
"ABC"""" D"
"ABC][ sd [dfd]"
"ABC''''dd"
```
Just to make sure we have covered all the corner cases.

Reply:
Along with multi byte character case

Reply:
Resolved.